feature/selector.py

```diff
@@ -166,6 +167,28 @@ def _validate(self):
             check_true(isinstance(self.alpha, (int, float)), TypeError("Alpha must a number."))
             check_true(self.alpha >= 0, ValueError("Alpha cannot be negative"))
 
+    class KL_Divergence(NamedTuple):
+        """
+
+        Computes the distribution of a given feature for instances where label == 1 and label == 0.
+        Uses KL divergence between the two distributions as an importance score,
+        where a higher value indicates greater discriminative power of the feature
+        with respect to the binary label.
+
```

Comment:
Do you want to add a comment on complexity of the method? For how many rows (observations) is this typically supposed to be run? OR when does it become a bottleneck, roughly speaking. I am looking for sth like, run this for less than 10K rows, or 100K rows Sth like that